assets/js/components/PostSearcher.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * PostSearcher component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useState, useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Button from './button';
+import Layout from './layout/layout';
+import { STORE_NAME as CORE_MODULES } from '../googlesitekit/modules/datastore/constants';
+import { STORE_NAME as CORE_SITE } from '../googlesitekit/datastore/site/constants';
+import PostSearcherAutoSuggest from './PostSearcherAutoSuggest';
+
+const { useSelect } = Data;
+
+function PostSearcher() {
+	const [ canSubmit, setCanSubmit ] = useState( false );
+	const [ match, setMatch ] = useState( {} );
+	const analyticsModuleActive = useSelect( ( select ) => select( CORE_MODULES ).isModuleActive( 'analytics' ) );
+
+	const adminURL = useSelect( ( select ) => {
+		const args = {};
+
+		if ( match && match.ID ) {
+			const {
+				ID: id,
+				permalink: permaLink,
+				post_title: pageTitle,
+			} = match;
+
+			args.id = id;
+			args.permaLink = permaLink;
+			args.pageTitle = pageTitle;
```

Comment:
`permaLink` is the only parameter that is used here now that #174 is done, so `id` and `pageTitle` can be removed.